api-audit/src/main/java/com/capitalone/dashboard/evaluator/CodeQualityEvaluator.java

```diff
@@ -123,7 +123,7 @@ private CodeQualityAuditResponse getStaticAnalysisResponse(CollectorItem collect
             if (metric.getName().equalsIgnoreCase("quality_gate_details")) {
                 codeQualityAuditResponse.addAuditStatus(CodeQualityAuditStatus.CODE_QUALITY_GATES_FOUND);
                 try {
-                    JSONObject qualityGateDetails = (JSONObject) new JSONParser().parse(metric.getValue().toString());
```

Comment:
create a separate PR for bug fixes

Reply:
This change is removed from this PR. And this is taken care in another SCA PR